package.json

```diff
@@ -51,11 +51,6 @@
       "lib/**"
     ]
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
```

Comment:
Please add this back.

Reply:
I will -- I just have an apparently weird set up where it can't find my node exe and I'm too lazy to do anything about it right now :)

Reply:
add nodejs to your path 😕

Reply:
'tis.

Reply:
well it is meant to fix your staged commits, so if you start committing code that is not formatted it becomes a problem 😓

Reply:
Fair enough -- re-added. Turned out my env wasn't so strange after all.